src/main/java/com/externalSorting/app/ExternalSortingRunner.java

```diff
@@ -35,21 +55,21 @@
         List<String> buffer = new ArrayList<>();
         List<File> files = new ArrayList<>();
 
-        try (BufferedReader reader = Files.newBufferedReader(Paths.get(filePath))) {
-            int count = 0;
+        try (BufferedReader reader = new MyBufferedReader(filePath)) {
+            int currentEstimatedFileSize = 0;
             for (String line = reader.readLine(); line != null; line = reader.readLine()){
                 List<String> splitLine = Arrays.asList(formatString(line));
-                count++;
-                // splitLine.removeIf(word -> word.isEmpty());
+                for (String s: splitLine) {
+                    currentEstimatedFileSize += s.length() + 1;
```

Comment:
You can use `getBytes()` method to know exactly how much space the `s` line takes.